### PR TITLE
pseudo spelling correction

### PR DESCRIPTION
**Pull request by dragthor**
http://www.thefreedictionary.com/pseudo

### CODE REVIEW

File: dist/js/foundation.js [hunk 1]
```diff
@@ -1323,7 +1323,7 @@
 	});
 })(jQuery);
 /****************************************************
- * Method for adding psuedo drag events to elements *
+ * Method for adding pseudo drag events to elements *
```
Comment (IamManchanda) [inline dist/js/foundation.js]: Dist files are automatically generated 
You should be editing source files instead 

Here are the files/lines where you should edit,
=> https://github.com/zurb/foundation-sites/blob/develop/js/foundation.util.touch.js#L79
=> https://github.com/zurb/foundation-sites/blob/develop/js/foundation.sticky.js#L108
=> https://github.com/zurb/foundation-sites/blob/develop/js/foundation.sticky.js#L149

**Review by IamManchanda — CHANGES_REQUESTED** [review]

**Comment by dragthor** [comment]
Yeah, I know.  `dist` is under version control.  Maybe they should `.gitignore` it?

**Comment by IamManchanda** [comment]
no,, not at all ... what about those who download css/js only version instead of compiled one ??

**Comment by kball** [comment]
Dist files are updated at release time, and we dont' want them to change between releases... happy to take the comment updates for the sourcefiles alone if you can update.

**Comment by IamManchanda** [comment]
@kball as @dragthor didn't reverted .... i did it .... please merge it!

**Comment by kball** [comment]
👍

**Pull request merged**